doc/installation.rst

```diff
@@ -252,7 +247,7 @@ Next, you must configure the EOS build using the ``configure`` script.
 To use the EOS Python interface you must pass ``--enable-python`` to the call ``configure``.
 The default is \cli{--disable-python}.
```

Comment:
While you are at it, can you fix the LaTeX macro used here? It should be ``--disable-python``